embedchain/vectordb/chroma_db.py

```diff
@@ -39,9 +39,11 @@ def _get_or_create_db(self):
         """Get or create the database."""
         return self.client
 
-    def _get_or_create_collection(self):
+    def _get_or_create_collection(self, name=None):
```

Comment:
then I think this is redundant, isn't it?

Reply:
Sure, but then I have to delete line 6 in `base_vector_db.py`.